Tools/TestWebKitAPI/Tests/WebKitObjC/CustomProtocolsInvalidScheme.mm

```diff
@@ -74,8 +80,10 @@ - (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigati
     webView.get().navigationDelegate = loadDelegate.get();
 
     [webView loadRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:@"ht'tp://www.webkit.org"]]];
+    Util::runFor(200_ms);
```

Comment:
This 200_ms wait seems gross; it would be nice if we could avoid this arbitrary waiting, given how slow our tests run already.

Reply:
I can reduce the timeout. Sadly, I don't believe there is currently a better way. No delegate calls happens currently, since the load doesn't even attempt navigation policy decision with the client.